python_modules/libraries/dagster-databricks/dagster_databricks/types.py

```diff
@@ -49,14 +49,14 @@ class DatabricksRunState(NamedTuple):
 
     def has_terminated(self) -> bool:
         """Has the job terminated?"""
-        return self.life_cycle_state.has_terminated()  # type: ignore  # (possible none)
+        return self.life_cycle_state is not None and self.life_cycle_state.has_terminated()
```

Comment:
`is not None` doesn't need to be explicit, but up to you

```suggestion
        return self.life_cycle_state and self.life_cycle_state.has_terminated()
```

Reply:
If you want a boolean return then you need `is not None` or to wrap in `bool(...)`, I prefer the former